notification-portlet-webapp/src/main/webapp/scripts/job-postings.js

```diff
@@ -150,14 +138,14 @@ var jobPostings = function(){
         /**
          * Modal Button Event handlers
          */
-        $('#' + settings.portletId + ' .emailFriendButton').on('click', function(e) {
+        $('#' + settings.portletId + ' .emailFriendButton').on('click', function() {
             $('#' + settings.portletId + ' .modal-overlay').show();
         });
-        $('#' + settings.portletId + ' .cancelEmailButton').on('click', function(e) {
+        $('#' + settings.portletId + ' .cancelEmailButton').on('click', function() {
             $('#' + settings.portletId + ' .modal-overlay').hide();
         });
-        $('#' + settings.portletId + ' .sendEmailButton').on('click', function(e) {
-            $.post( "test.php", $('#' + settings.portletId + ' .emailFriendForm').serialize());
+        $('#' + settings.portletId + ' .sendEmailButton').on('click', function() {
+            $.post( 'test.php', $('#' + settings.portletId + ' .emailFriendForm').serialize());
```

Comment:
Its also somewhat curious that there is a call to `test.php` in a portlet with no PHP code.